packages/app/src/server/models/attachment.js

```diff
@@ -34,6 +36,11 @@ module.exports = function(crowi) {
     fileSize: { type: Number, default: 0 },
     temporaryUrlCached: { type: String },
     temporaryUrlExpiredAt: { type: Date },
+    attachmentType: {
+      type: String,
+      enum: AttachmentType,
+      default: AttachmentType.WIKI_PAGE,
```

Comment:
You don't need this.
Add `required: true` instead.

One of the `AttachmentType`s MUST BE PROVIDED when uploading any kinds of attachments.